util/flags/manager.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
2024

Reply:
This file was technically renamed and merged with another file, does this require bumping the copyright header?

Reply:
nope.

Reply:
kk, done. Previously we only added the current year when we added new files. But I'm in general not aware of the rules apart from that :)